### DISCUSSION TITLE
Connection pool metrics

### DISCUSSION THREAD

**Discussion by jianzhou520** [prisma/prisma#21523] [Q&A]
### Question

Currently, our team is facing QPS increase fast, so we want to tune the connection pool metrics. From prisma metrics, we find some shortcuts unnormal.
![image](https://github.com/prisma/prisma/assets/11852997/7d814ebb-3147-4ed7-8bf1-ee8a01f7d834)
We print logs every 10 minitunes, it shows when QPS high, the pool_wait_count is high but the pool_idle_connections is not empty. It looks like the connection pool has enough idle connections but not to handle wait connections. So we want to know weather the idle connection is truly the idle that can handle wait connections directly, or some idle connections maybe in releasing progress?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
`metrics` is a preview feature, so it can very much be that some of the returned data is not fully correct yet. If you think that the data that is being returned is not correct, please open a bug report issue - optimally with a reproduction that can show us what is wrong. Thanks!

What version of Prisma re you using?
You write you print logs every 10 minutes, but your screenshot shows much more frequent logging. Did I misunderstand?
What exactly am I seeing in the screenshot?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
